pkg/core/storage/boltdb_store.go

```diff
@@ -100,17 +99,15 @@ func (s *BoltDBStore) PutBatch(batch Batch) error {
 
 // Seek implements the Store interface.
 func (s *BoltDBStore) Seek(key []byte, f func(k, v []byte)) {
-	err := s.db.View(func(tx *bbolt.Tx) error {
+	// no error will be returns as inner function returns no error
```

Comment:
I wouldn't be so sure about that, you don't know what kind of error `View` itself can return. Just look at the implementation: https://github.com/etcd-io/bbolt/blob/master/db.go#L708